docs/monitoring.md

```diff
@@ -32,3 +34,43 @@ Limitations:
 - An index pattern `logstash-*` may need to be create on the first access
 - The ElasticSearch server is not clustered and has limited querying capacity. It can be easily overloaded.
 - Logs are kept for two weeks.
+
+
+## IDR FTP Monitoring
+
+The IDR FTP server has limited Prometheus monitoring running on the same server with alerts for low available disk space.
+
+
+## IDR Pilot Monitoring
+
+All IDR pilot test instances are currently monitored by a single monitoring server.
+This is accessed using the same commands as above, using `idr-pilot` as the proxy.
```

Comment:
Might be better to be explicit since there's the "centralized logged" between this and the first addition. (Or move the logging to the bottom?)